.github/workflows/pull-request.yml

```
@@ -288,33 +288,33 @@
       
-  #     - name: Clone API Repository
-  #       run: |
-  #         # Retrieve the complete branch name directly from the GitHub context
-  #         FULL_BRANCH_NAME=${{ github.base_ref }}
-  #         echo "FULL_Branch_NAME: $FULL_BRANCH_NAME"
+      - name: Clone API Repository
+        run: |
+          # Retrieve the complete branch name directly from the GitHub context
+          FULL_BRANCH_NAME=${{ github.base_ref }}
+          echo "FULL_Branch_NAME: $FULL_BRANCH_NAME"
           
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)